app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,12 @@
 
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
+    <% if ENV["GOOGLE_ANALYTICS_ID"] %>
+
+      <%= javascript_include_tag "https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_ANALYTICS_ID"]}" %>
```

Comment:
Does this line belong here if we're not using Tag Manager?

Reply:
It's what Google say to "copy and paste" into the site to enable Google Analytics, along with the code below.

Reply:
Ah ok. Misled by the name.

Reply:
Yeah, it's confusing, but it looks like even without using full Tag Manager, Google Analytics is still loaded using this JS.